spec/unit/bricks/middleware/logger_middleware_spec.rb

```diff
@@ -52,4 +52,36 @@
       end
     end
   end
+
+  context 'when COLLECT_STATS parameter set to true' do
+    let(:params) { { 'COLLECT_STATS' => 'true' } }
+    let(:app) { double(:app) }
+
+    before do
+      subject.app = app
+      allow(app).to receive(:call)
+    end
+
+    it 'turns splunk logging on' do
+      expect(GoodData).to receive(:logging_splunk_on)
+      expect(GoodData.splunk_logger).not_to be_an_instance_of(GoodData::SplunkLogger)
+      subject.call(params)
+    end
+  end
+
+  context 'when COLLECT_STATS parameter set to false' do
```

Comment:
Similar code found in logger_middleware_spec.rb:56, logger_middleware_spec.rb:72 (mass = 90)